app/assets/stylesheets/custom.scss

```
@@ -1,6 +1,13 @@
 @import 'bootstrap-sprockets';
 @import 'bootstrap';
 @import 'color.scss';
+/* maxins, variables, etc. */
+@mixin box_sizing {
+  -moz-box-sizing:    border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing:         border-box;
```

```suggestion
  box-sizing: border-box;
```
just use 1 space after `:`